uber/templates/registration/checkin.html

```diff
@@ -0,0 +1,117 @@
+<script type="text/javascript">
+    $(function() {
+        $(".dialog-form").dialog({ autoOpen: false });
+
+        $(".attendee-checkin").button().on("click", function () {
+        attendee_id = $(this).attr('data-attendeeid');
+
+        dialog = $("#" + attendee_id).dialog({
+            autoOpen: false,
+            height: 450,
+            width: 500,
+            modal: true,
+            buttons: {
+                "Save & Check In": function() {
+                    check_in(attendee_id);
+                    dialog.dialog("close");
+                },
+                Cancel: function () {
+                    dialog.dialog("close");
+                }
+            },
+            // FIXME: OMG TERRIBLE POSITION HAX
+            open: function () {
+              $('.ui-dialog').css("position","fixed")
+                .css("top","5%")
+                .css("left","33%");
+            },
+            close: function () {
+                form[ 0 ].reset();
+            }
+        });
+
+        form = dialog.find("form").on("submit", function (event) {
+            event.preventDefault();
+            check_in(attendee_id);
+            dialog.dialog("close");
+        });
+
+            dialog.dialog("open");
+        });
+    });
+
+    function check_in(id) {
```

Comment:
Checking in doesn't seem to be working; I get the Javascript error ``Uncaught TypeError: Cannot read property 'dialog' of undefined``.

Looks like this might need some work.  Working on it now.